src/scene/container/container-mixins/effectsMixin.ts

```diff
@@ -9,7 +9,7 @@ import type { Effect } from '../Effect';
 
 export interface EffectsMixinConstructor
 {
-    mask?: number | Container;
+    mask?: number | Container | null | undefined;
```

Comment:
Being an optional type it implicitly accepts `undefined` so you just need to add `null`.
```suggestion
    mask?: number | Container | null;
```